common/config/src/locale/ta-IN.ts

```diff
@@ -0,0 +1,173 @@
+import type { DeepPartial } from '../types'
+
+export interface LocaleConfig {
```

Comment:
Well, I missed here, you can refer the `en-US.ts` to fix here.

```suggestion
import { defineLocaleConfig } from './helper'

export function taINLocale() {
  return defineLocaleConfig({
```